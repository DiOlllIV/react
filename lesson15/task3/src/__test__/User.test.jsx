import React from 'react'
import {shallow} from 'enzyme'
import User from '../User'

describe('<User/>', () => {
  it('should return user with name James and Age 28', () => {
    const user = {
        name: "James",
        age: 28,
    }
    const wrappedComponent = shallow(<User name={user.name} age={user.age} />);

    expect(wrappedComponent).toMatchSnapshot();
  });
});
