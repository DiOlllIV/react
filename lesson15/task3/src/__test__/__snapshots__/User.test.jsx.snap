// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<User/> should return user with name James and Age 28 1`] = `
<li
  className="user"
>
  <span
    className="user__name"
  >
    James
  </span>
  <span
    className="user__age"
  >
    28
  </span>
</li>
`;
